code/game/objects/items/melee/misc.dm

```diff
@@ -326,6 +326,16 @@
 	cooldown = 10
 	stamina_damage = 20
 
+//Wooden Baton
+/obj/item/melee/classic_baton/police/beater
+	name = "wooden baton"
+	desc = "Favored by hot headed Security Officers who don't want to get in trouble with CentCom but still want to get that nostalgic feeling of beating some criminal scum upside the head with a chunk of wood."
+	force = 8
+	throwforce = 7
+	cooldown = 0
```

Comment:
https://github.com/BeeStation/BeeStation-Hornet/assets/39193182/d4ab72e6-bfe1-4f55-88b6-80fb28b7ba7f

https://github.com/BeeStation/BeeStation-Hornet/assets/39193182/0b10573e-b0c3-41b1-8618-b58e5f107c2d

Reply:
This has been adressed in my last changes! Spent so long on this!